NEWS

```diff
@@ -4,7 +4,7 @@ Version 0.6.1 [WIP]
 DEVELOPMENT
 * The vignette, and documentation of the maxmind function, have been improved to make explicit
 the datasets rgeolocate ships with (#25)
-* Added ability to retrieve city_geoname_id from maxmind databases
+* Added feature to retrieve city_geoname_id from maxmind databases
```

Comment:
Ah, miscommunication; I meant that there should be a features section for newly-shipped features. Oops ;p.